openpype/hosts/maya/plugins/publish/extract_animation.py

```diff
@@ -23,6 +23,14 @@ class ExtractAnimation(openpype.api.Extractor):
     families = ["animation"]
 
     def process(self, instance):
+        if instance.data.get("farm"):
+            path = os.path.join(
+                os.path.dirname(instance.context.data["currentFile"]),
+                "cache",
+                instance.data["name"] + ".abc"
+            )
+            instance.data["expectedFiles"] = [os.path.normpath(path)]
+            return
```

Comment:
Some questions:
1. What's this needed for? Why do we need to set `expectedFiles`?
2. The path here wouldn't be correct, right? The farm would still extract to a temporary staging dir, no? Or did I miss the code that influences staging dir on the farm?

Reply:
Yeah, you are right, removed.